src/lib/services/setting-service.ts

```diff
@@ -29,8 +29,16 @@ export default class SettingService {
         this.eventService = eventService;
     }
 
-    async get<T>(id: string, defaultValue?: T): Promise<T> {
```

Comment:
Type was wrong, so we added a new method with the right type called getWithDefault to avoid making this a breaking change